python.rst

```diff
@@ -108,14 +165,39 @@ Good
    class ExampleService(geordi_base.OscarServiceBase):
        pass
 
+
+
+- When you write new classes or modules, make sure to name them in a
+  way that reads well when imported this way, instead of being redundant.
```

Comment:
Feels like minor improvements to readability aren't worth sacrificing discoverability.

Reply:
how are they in conflict here?

Reply:
this suggests removing context from the file name about what's in the file.

Reply:
ahhh, I see. That's a fault of my example. Would you consider `kafka_consumer.Batched` a good name?